### PR TITLE
CIT-31 #time 6h Create models and relations between models , fetching…

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
… data from database for templates in the blog app in myWeb project and update models in the bookOutlet app in Book_store project

### CODE REVIEW

File: Projects/Book_store/bookStore/bookOutlet/models.py [hunk 1]
```diff
@@ -2,16 +2,41 @@
 
 # Create your models here.
 
+class Country(models.Model):
+    name= models.CharField(max_length=50)
+
+
+class Address(models.Model):
+    country = models.CharField(max_length=50)
```
Comment (arvish-codal) [inline Projects/Book_store/bookStore/bookOutlet/models.py]: @codal-uchoudhary , shouldn't this be a ForeignKey ? You have already created Country model above at line no 5.

File: Projects/My_website/myWebsite/blog/models.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Post(models.Model):
+    title=models.CharField(max_length=100)
+    excerpt = models.CharField(max_length=200)
+    image_name = models.CharField(max_length=30)
+    data = models.DateField(auto_now=True)
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/blog/models.py]: Pls rename `data` field to `created_at`.

File: Projects/My_website/myWebsite/blog/views.py [hunk 3]
```diff
@@ -1,75 +1,20 @@
...
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis nobis
-          aperiam est praesentium, quos iste consequuntur omnis exercitationem quam
-          velit labore vero culpa ad mollitia? Quis architecto ipsam nemo. Odio.
-        """
-    }
-]
 
 def startingPage(request):
-    return render(request,'blog/index.html',{"posts":allPosts})
+    latest_posts = Post.objects.all().order_by('-data')[:3]
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/blog/views.py]: @codal-uchoudhary , Make changes here as well after you rename the date field.

File: Projects/Book_store/bookStore/bookOutlet/models.py (hunk 1 above)
Comment (codal-uchoudhary) [reply Projects/Book_store/bookStore/bookOutlet/models.py]: we can , but here  Address model refers to the address of Author , so i created one-to-one relation between the Author and Address model. 
and , Country model refers to the countries in which a particular book is published so i created many-to-many relation between the Country and Book model.

... for practice purpose let's do this change.

**Comment by codal-uchoudhary** [comment]
All changes are done 
- foreign key is added in Address model in bookOutlet app
- data is changed to data in Blog app

**Review by arvish-codal — APPROVED** [review]

**Review by arvish-codal — APPROVED** [review]

**Pull request merged**